mscore/uploadscoredialog.ui

```diff
@@ -129,6 +126,9 @@
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
+         <property name="sizeAdjustPolicy">
+          <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>

```

Comment:
Maybe this and the other could get set programatically, only if the screen is not high enough?